### ISSUE TITLE
In-code documentation provides invalid command

### ISSUE THREAD

**Issue by evanmiller2112** [MythicMeta/Mythic_CLI#5]
Relatively minor documentation thing:

`iditabad@ubuntu:~/Mythic$ sudo ./mythic-cli status
CONTAINER NAME		MYTHIC SERVICE		WEB ADDRESS		BOUND LOCALLY
mythic_nginx		Nginx (Mythic Web UI)	https://127.0.0.1:7443	 false
< ... snip ... >
[*] If you are using a remote PayloadType or C2Profile, they will need certain environment variables to properly connect to Mythic.
    Use 'sudo ./mythic-cli payload config' or '**sudo ./mythic-cli c2 config**' for easy-to-use configs for these services.
`

the two commands in question are:
`sudo ./mythic-cli payload config`
and
`sudo ./mythic-cli c2 config`

Basically it should be `sudo ./mythic-cli config [payload|c2]`

The actual command is `sudo ./mythic-cli config c2`. Neither the "c2" nor the "payload" commands have "config" subcommands. option, and result in:

`iditabad@ubuntu:~/Mythic$ sudo ./mythic-cli c2 config
2022/04/21 09:33:51 [-] Unknown subcommand: config
iditabad@ubuntu:~/Mythic$ sudo ./mythic-cli payload config
2022/04/21 09:34:28 [-] Unknown subcommand: config`

**Comment by its-a-feature** [comment]
Should be addressed with latest commit: https://github.com/MythicMeta/Mythic_CLI/commit/3e838b6b0bd9feb55ff48c11d5f1e0dd09824086
